cmd/controller-manager/app/options/feature_options.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+	genericcontrollermanager "k8s.io/kubernetes/cmd/controller-manager/app"
+)
+
+type FeatureOptions struct {
```

Comment:
these sound more like Debugging options. Features is a pretty neutral term. Moreover, we have something called "feature gates" in kube. So call this Feature might be confusing.

Reply:
done